cms/djangoapps/contentstore/utils.py

```diff
@@ -81,20 +83,26 @@ def delete_course_and_groups(course_key, user_id):
 
 def get_lms_link_for_item(location, preview=False):
     """
-    Returns an LMS link to the course with a jump_to to the provided location.
+    Returns an LMS link to the course with a jump_to to the provided location, also checks if
+    Comprehensive Theming is enabled return the LMS link of enabled site's domain name.
 
     :param location: the location to jump to
     :param preview: True if the preview version of LMS should be returned. Default value is false.
     """
     assert isinstance(location, UsageKey)
 
-    if settings.LMS_BASE is None:
-        return None
+    lms_base = None
+    if theming_helpers.is_comprehensive_theming_enabled():
```

Comment:
No need to check if comprehensive theming is enabled. We only care if there is a SiteConfiguration associated with the org for this item.